rpc/websockets.go

```diff
@@ -411,9 +413,9 @@ type Header struct {
 }
 
 func (api *pubSubAPI) subscribeNewHeads(wsConn *wsConn, subID rpc.ID) (context.CancelFunc, error) {
-	ctx, cancel := context.WithCancel(context.Background())
+	_, cancel := context.WithCancel(context.Background())
```

Comment:
This cancel function is obsolete as the context is not used anywhere.

Reply:
cancel func needed for eth_unsubscribe handling
I changed the code for this function

Reply:
thanks, can you also sign the commits

Reply:
I signed the commits and added websocket error logging